### PR TITLE
WIP: Redirect crash reports to a "crashreports" channel

### PR DESCRIPTION
**Pull request by Anarchid**
Should work but coded blindly and untested.

### CODE REVIEW

File: Zero-K.info/Controllers/GithubController.cs [hunk 1]
```diff
@@ -32,13 +32,17 @@ public ActionResult Hook()
...
+            string channel = "zkdev";
             Object[] values;
 
             switch (eventType) {
                 case "issues":
-                    if (payload.action == "labeled")
+                    if (payload.action == "labeled"){
                         break;
+                    }
+                    if (payload.repository.name="CrashReports"){
```
Comment (sprunk) [inline Zero-K.info/Controllers/GithubController.cs]:
```suggestion
                    if (payload.repository.name=="CrashReports"){
```

Comment (sprunk) [reply Zero-K.info/Controllers/GithubController.cs]: >should work

Comment (Anarchid) [reply Zero-K.info/Controllers/GithubController.cs]: Oops

Comment (sprunk) [reply Zero-K.info/Controllers/GithubController.cs]: story of my life

Comment (Anarchid) [reply Zero-K.info/Controllers/GithubController.cs]: should work *now*!

**Pull request merged**